lib/src/Makefile.am

```diff
@@ -53,4 +53,6 @@ tests_SOURCES = tests.cpp
 tests_CPPFLAGS = $(DBGFLAGS) $(AM_CPPFLAGS) $(CFLAGS_COMMON)
 tests_LDADD = -lhiredis -lswsscommon -lpthread -L$(top_srcdir)/lib/src/.libs -lsairedis -L$(top_srcdir)/meta/.libs -lsaimetadata -lsaimeta
 
+if RTEST
```

Comment:
is this disabled by default ?

Reply:
Yes, you may check configure.ac

---
In reply to: [141856302](https://github.com/Azure/sonic-sairedis/pull/233#discussion_r141856302) [](ancestors = 141856302)